docs/pages/includes/install-windows.mdx

```diff
@@ -1,23 +1,24 @@
 Most `tsh` features are supported for Windows 10 1607+. The `tsh ssh` command
 can be run under `cmd.exe`, PowerShell, and Windows Terminal.
 
-To install `tsh` on Windows, run the following commands in **PowerShell** (these commands will not work in `cmd.exe`):
+To install `tsh` and `tctl` on Windows, run the following commands in
+**PowerShell** (these commands will not work in `cmd.exe`):
 
-<Tabs dropdownView dropdownCaption="Teleport Edition">
+<Tabs>
```

Comment:
Removed the `dropdownView` here because the docs page felt very odd with macOS showing tabs and Windows (right after it) showing a dropdown:

<img width="942" alt="image" src="https://github.com/gravitational/teleport/assets/7527103/fd2e7e22-ac28-4f0a-a07a-c2717bb489d6">